### DISCUSSION TITLE
Bookmarks: Link spacing causes links to not be horizontally centred

### DISCUSSION THREAD

**Discussion by j-freddy** [bluecaret/carettab#348] [Help and Support]
Adding spacing between links causes links to not be horizontally centred (see images below). This becomes more noticeable if the spacing is big.

<img width="640" alt="bar" src="https://github.com/bluecaret/carettab/assets/76404803/a927fb01-8903-442f-b29a-50b933645be9">
<img width="640" alt="baz" src="https://github.com/bluecaret/carettab/assets/76404803/5d5196b8-789c-4924-b4cd-73cc3730aa2e">

The user can currently fix this by setting "Space Between Links" to 0, increasing the "Padding around link text" then adjusting the "Window alignment and position".

<img width="640" alt="foo" src="https://github.com/bluecaret/carettab/assets/76404803/9b2d27cc-dd8b-4aff-b4ee-237cf2f39baf">

**Comment by bluecaret** [comment]
Very good catch! Thank you for reporting, I'll try to get that resolved.
